packages/flutter_tools/test/application_package_test.dart

```diff
@@ -20,13 +20,31 @@ import 'src/context.dart';
 
 void main() {
   group('ApkManifestData', () {
-    test('Select explicity enabled activity', () {
+    test('Avoid selecting only enabled, but not launcher&main activity', () {
```

Comment:
Please name all tests in the affirmative instead of the negative.  For example:
'A main launcher Activity is chosen even when an enabled Activity is available'